frontend/wallet/src/render/components/TransactionCell.re

```diff
@@ -322,17 +334,66 @@ module InfoSection = {
   };
 };
 
+// Making a separate styles module here to collocate the styles in the below
+// make function closer to the layout
+module TopLevelStyles = {
+  open Css;
+
+  module Actors = {
+    let wrapper = style([padding2(~h=`zero, ~v=`rem(0.625))]);
+
+    let mode = style([marginTop(`rem(0.25))]);
+  };
+
+  let infoSection =
+    style([
+      verticalAlign(`baseline),
+      width(`percent(100.)),
+      height(`calc((`sub, `percent(100.0), `rem(0.75)))),
```

Comment:
Is this to account for the row containing status/date being above the memo? Any reason we wouldn't want to structure it like this?
```html
<div flex direction:column>
  <StatusAndDate flex justifyContent:end />
  <div flex direction:row>
    <Memo />
    <Amount />
  </div>
</div>
```

Reply:
nice good call

Reply:
oooh

Reply:
won't work because of the grid actually unless we make one grid cell span both memo and amount.

Reply:
Thats not hard to do with grid.

Reply:
nvm yeah brandon's right